src/lib/views/List.svelte

```diff
@@ -23,14 +23,14 @@
 <section>
 	<ul>
 		{#if offline}
-			<ErrorCard ErrorTitle={'Oeps, je bent nu offline!'} {ErrorMessage} />
+			<ErrorCard Title={'Oeps, je bent nu offline!'} {ErrorMessage} />
```

Comment:
Willen we props pascal cased hebben? Lijkt me ook wel slim om die dan consistent te doen.

Daarnaast.. `Cards/InfoCard.svelte` wordt nog geimporteerd, terwijl die niet meer hoort te bestaan. Klopt dat of staat die nu dubbel in version control